flytekit-local-engine/src/test/java/org/flyte/localengine/LocalEngineTest.java

```diff
@@ -556,18 +556,21 @@ public TestCaseExhaustivenessWorkflow() {
     }
 
     @Override
-    public void expand(SdkWorkflowBuilder builder) {
-      SdkBindingData<Long> x = builder.inputOfInteger("x");
+    public NoOpType expand(SdkWorkflowBuilder builder, NoOpType input) {
       SdkBindingData<Long> nextX =
           builder
               .apply(
                   "decide",
-                  when("eq_1", eq(ofInteger(1L), x), new NoOp(), NoOpType.create(x))
-                      .when("eq_2", eq(ofInteger(2L), x), new NoOp(), NoOpType.create(x)))
+                  when("eq_1", eq(ofInteger(1L), input.x()), new NoOp(), NoOpType.create(input.x()))
+                      .when(
+                          "eq_2",
+                          eq(ofInteger(2L), input.x()),
+                          new NoOp(),
+                          NoOpType.create(input.x())))
```

Comment:
With the change above we can revert this change
```suggestion
                  when("eq_1", eq(ofInteger(1L), x), new NoOp(), NoOpType.create(x))
                      .when("eq_2", eq(ofInteger(2L), x), new NoOp(), NoOpType.create(x)))
```